test/test_run_difference_with_gpao.py

```diff
@@ -98,11 +99,34 @@ def test_gpao_run_with_cog():
 
 
 @pytest.mark.gpao
-def test_gpao_run_without_cog():
+def test_gpao_run_without_cog_stream_rge():
     dtm_lidar_lhds = "./data/lhd_dir_gpao"
-    output_dir = TMP_PATH / "gpao_run_without_cog"
+    output_dir = TMP_PATH / "gpao_run_without_cog_stream_rge"
     output_dir.mkdir()
-    project_name = "test_run_altianalysis_gpao"
+    project_name = "test_run_altianalysis_gpao_stream"
+
+    gpao_hostname = os.environ.get("GPAO_API_URL", "localhost")
+    url_api = f"http://{gpao_hostname}:8080/api/"
+
+    runner_store_path = Path(dtm_lidar_lhds).resolve()
+    local_store_path = Path("data/lhd_dir_gpao").resolve()
+
+    run_difference_with_gpao.compute_on_gpao(
+        Path(dtm_lidar_lhds), None, Path(output_dir), gpao_hostname, local_store_path, runner_store_path, project_name
+    )
+
+    if gpao_hostname == "localhost":
+        tu.execute_gpao_client(tags="docker", num_thread=4)
+    wait_running_job(url_api, project_name, delay_second=1, delay_log_second=10)
+
+
+@pytest.mark.gpao
+def test_gpao_run_without_cog_given_secondary_folder():
+    dtm_lidar_lhds = "./data/lhd_dir_gpao"
+    secondary_dtm_dir = "./data/lhd_dir_gpao"
+    output_dir = TMP_PATH / "gpao_run"
```

Comment:
pour simplifier le fait de retrouver quel test a créé quel dossier je dirais plutôt 
```suggestion
    output_dir = TMP_PATH / "gpao_run_without_cog_given_secondary_folder"
```

Reply:
done